client/components/jetpack/card/jetpack-product-card/index.tsx

```diff
@@ -148,27 +148,26 @@ const JetpackProductCard: React.FC< OwnProps > = ( {
 						{ preventWidows( disabledMessage ) }
 					</p>
 				) }
-				{ ! isDisabled &&
-					( buttonURL ? (
-						<Button
-							primary={ buttonPrimary }
-							className="jetpack-product-card__button"
-							onClick={ onButtonClick }
-							href={ buttonURL }
-							disabled={ isDeprecated }
-						>
-							{ buttonLabel }
-						</Button>
-					) : (
-						<Button
-							primary={ buttonPrimary }
-							className="jetpack-product-card__button"
-							onClick={ onButtonClick }
-							disabled={ isDeprecated }
-						>
-							{ buttonLabel }
-						</Button>
-					) ) }
+				{ buttonURL ? (
+					<Button
+						primary={ buttonPrimary }
+						className="jetpack-product-card__button"
+						onClick={ onButtonClick }
+						href={ isDisabled ? '#' : buttonURL }
```

Comment:
What happens if we keep the `href` value intact when the button is disabled? Can users still click on it? In other words, why are we removing the URL when the button is disabled?

Reply:
Yes, that is the problem with `Button` component, if you pass `href`, disabled is irrelevant and it still becomes clickable.

Reply:
It makes sense. Thanks for the explanation.